packages/post-rector/src/NodeAnalyzer/NetteInjectDetector.php

```diff
@@ -45,8 +45,9 @@ private function isInjectPropertyAlreadyInTheClass(Class_ $class): bool
             if ($phpDocInfo === null) {
                 continue;
             }
+            $phpDocInfoGetTagsByName = $phpDocInfo->getTagsByName('inject');
 
-            if ($phpDocInfo->getTagsByName('inject') === []) {
```

Comment:
In many cases, you can re-use logic from https://github.com/rectorphp/rector/blob/master/docs/rector_rules_overview.md#renamevariabletomatchmethodcallreturntyperector

This service, to be specific:
https://github.com/rectorphp/rector/blob/5988584736f30bd3a43b2a65a0724d59d0504459/rules/naming/src/Rector/Assign/RenameVariableToMatchMethodCallReturnTypeRector.php#L155

<br>

The expected result here would be:

```bash
$phpDocTagNode = $phpDocInfo->getTagsByName('inject');

# or better
$injectPhpDocTagNode = $phpDocInfo->getTagsByName('inject');
```

Reply:
I will try.